precompiles/ArbRetryableTx_test.go

```diff
@@ -102,7 +103,15 @@ func testRetryableRedeem(t *testing.T, evm *vm.EVM, precompileCtx *Context) {
 	}
 }
 
-func TestRetryableRedeem(t *testing.T) {
+func overrideStateArbOSVersion(evm *vm.EVM, arbosVersion uint64) error {
```

Comment:
instead of that, use newMockEVMForTestingWithVersion

Reply:
I don't follw. If newMockEVMForTestingWithVersion needs to be fixed, let's fix it. Let's talk about it later.